### PR TITLE
PMM-6800 add ova staging

### PR DESCRIPTION
**Pull request by nikita-b**
I added job for starting staging based on OVA image

### CODE REVIEW

File: pmm/pmm2-ovf-staging-start.groovy [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                    slackSend botUser: true,
+                              channel: "@${OWNER_SLACK}",
+                              color: '#00FF00',
+                              message: "OVA staging for ${OVA_VERSION} was created. IP: https://${PUBLIC_IP}\nYou can stop instance here: ${BUILD_URL}/console"
+                }
+            }
+        }
+        stage('Waiting for ') {
+            input {
+                message 'Do you want to shutdown instance?'
```
Comment (puneet0191) [inline pmm/pmm2-ovf-staging-start.groovy:74]: how do we trigger this job from other jobs like upgrade etc if this needs manual intervention?

File: pmm/pmm2-ovf-staging-start.groovy [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            name: 'OVA_VERSION')
+    }
+    options {
+        skipDefaultCheckout()
+        buildDiscarder(logRotator(numToKeepStr: '30', artifactNumToKeepStr: '30'))
+        timeout(time: 1, unit: 'DAYS')
+    }
+    environment {
+        VM_NAME = "pmm-ovf-staging-${BUILD_ID}"
+        VM_MEMORY = "4096"
```
Comment (puneet0191) [inline pmm/pmm2-ovf-staging-start.groovy:30]: Possible to increase this?

**Review by puneet0191 — APPROVED** [review]

File: pmm/pmm2-ovf-staging-start.groovy [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import hudson.model.Node.Mode
```
Comment (atymchuk) [inline pmm/pmm2-ovf-staging-start.groovy:1]: Can we leave a short comment at the beginning of each pipeline to understand its purpose. Question: will this stage a VM on DigitalOcean? Let's then mention it there.

File: pmm/pmm2-ovf-staging-start.groovy [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+                        def OWNER_SLACK = slackUserIdFromEmail(
+                            botUser: true,
+                            email: BUILD_USER_EMAIL,
+                            tokenCredentialId: 'JenkinsCI-SlackBot-v2'
+                        )
+
+                        slackSend botUser: true,
+                                channel: "@${OWNER_SLACK}",
+                                color: '#00FF00',
+                                message: "OVA staging for ${OVA_VERSION} was created. IP: https://${PUBLIC_IP}\nYou can stop instance here: https://pmm.cd.percona.com/job/pmm2-ovf-staging-stop/build"
```
Comment (atymchuk) [inline pmm/pmm2-ovf-staging-start.groovy:103]:
```suggestion
                                message: "OVF instance for ${OVA_VERSION} was created. IP: https://${PUBLIC_IP}\nYou can stop it with: https://pmm.cd.percona.com/job/pmm2-ovf-staging-stop/build"
```

File: pmm/pmm2-ovf-staging-stop.groovy [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+]) _
+
+pipeline {
+    agent {
+        label 'cli'
+    }
+    parameters {
+        string(
+            defaultValue: 'ALL',
+            description: 'Name or IP of VM to stop. ALL means stop all vm older 24 hours',
```
Comment (atymchuk) [inline pmm/pmm2-ovf-staging-stop.groovy:13]:
```suggestion
            description: 'Name or IP of VM to stop. ALL means stop all VMs older than 24 hours',
```

**Review by atymchuk — APPROVED** [review]

**Pull request merged**